packages/fxa-payments-server/.sass-lint.yml

```diff
@@ -0,0 +1,33 @@
+files:
+  include: 'src/**/*.scss'
```

Comment:
This `.sass-lint.yml` is copied over from `fxa-content-server` with just the file glob here changed.